src/components/orders/DetailsTable/index.tsx

```diff
@@ -170,11 +211,16 @@ export function DetailsTable(props: Props): JSX.Element | null {
                 {areTradesLoading ? (
                   <Spinner />
                 ) : txHash ? (
-                  <RowWithCopyButton
-                    textToCopy={txHash}
-                    onCopy={(): void => onCopy('settlementTx')}
-                    contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
-                  />
+                  <Wrapper>
+                    <RowWithCopyButton
+                      textToCopy={txHash}
+                      onCopy={(): void => onCopy('settlementTx')}
+                      contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
+                    />
+                    <Button role="button">
```

Comment:
This should trigger a route change so it makes sense for it be a Link and it should point to `to={`/tx/${txHash}/batch-graph`}`

Reply:
Update: url according to https://github.com/cowprotocol/explorer/pull/49 is `/tx/${txHash}/?tab=graph`